templates/definition/vehicle/iso15118.yaml

```diff
@@ -0,0 +1,76 @@
+template: iso15118
+products:
+  - description:
+      generic: ISO15118
+group: generic
+requirements:
+  description:
+    de: Nur unterstützt wenn das Fahrzeug den Ladestand (Soc) an die verbundene Wallbox übermitteln kann.
+    en: Only supported if the vehicle can provided the state of charge (Soc) to the connected wallbox.
+params:
+  - name: title
+  - name: icon
+    default: car
+    advanced: true
+  - name: capacity
+  - name: phases
+    advanced: true
+  - name: mode
+    advanced: true
+  - name: minCurrent
+    advanced: true
+  - name: maxCurrent
+    advanced: true
+  - name: identifiers
+    advanced: true
+    type: stringlist
+  - name: minSoc
+    advanced: true
+    default: 0
+    help:
+      de: Ladung mit maximaler Geschwindigkeit bis zu dem angegeben Ladestand unabhängig PV-Erzeugung, wenn der Lademodus nicht auf 'Aus' steht.
+      en: Immediate charging with maximum power up to the defined state of charge independently from solar production if the charge mode is not set to 'OFF'.
```

Comment:
Warum ist das Off in Uppercase? In der UI ist es ja wie das deutsche nur der erste Buchstabe groß.

Reply:
Fix für default in https://github.com/evcc-io/evcc/pull/8312

Reply:
Das wurde aus den Defaults kopiert. Ist hier jetzt sowieso entfernt.

Reply:
Wie wäre es mit `"vehicle-identify"`?